tests/Feature/Connect/LeaderboardInfoTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Connect;
+
+use App\Models\Game;
+use App\Models\Leaderboard;
+use App\Models\System;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+use Tests\TestCase;
+
+class LeaderboardInfoTest extends TestCase
+{
+    use BootstrapsConnect;
+    use RefreshDatabase;
+
+    public function testLeaderboardInfo(): void
+    {
+        $now = Carbon::now();
+        Carbon::setTestNow($now);
+
+        /** @var System $system */
+        $system = System::factory()->create();
+        /** @var Game $game */
+        $game = Game::factory()->create(['ConsoleID' => $system->id]);
+        /** @var Leaderboard $leaderboard */
+        $leaderboard = Leaderboard::factory()->create(['GameID' => $game->id]);
+
+        $this->get($this->apiUrl('lbinfo', ['i' => $leaderboard->id]))
+            ->assertStatus(200)
+            ->assertExactJson([
+                'Success' => true,
+                'LeaderboardData' => [
+                    'ConsoleID' => $system->id,
+                    'ConsoleName' => $system->name,
+                    'Entries' => [],
```

Comment:
I'd like to see a second test that validates Entries. It's good to have an empty test and a non-empty test.

Ideally, we'd also have a test where Entries is truncated, and one where the `u` parameter is passed, but I understand that this is is just "basic coverage".

Reply:
No problem - this coverage was mostly a reaction to the formatting issue that came up a few weeks ago (ie: `'LBCreated' => $now->format('Y-m-d H:i:s')`).

However, adding another test for entries was no big deal. Added in latest.